.github/workflows/pyinstaller_gui_mac.yaml

```diff
@@ -1,25 +1,37 @@
 name: Compile GUI for Mac
 # run this job manually
 on: workflow_dispatch
```

Comment:
need to run the actions when we create and tag a new release
`on:
  push:
    # Pattern matched against refs/tags
    tags:        
      - '**'           # Push events to every tag including hierarchical tags like v1.0/beta
`